src/business.ts

```diff
@@ -168,4 +172,42 @@ export class BusinessApi {
     }
     return { success: !resp?.payload?.errorCode, body: { ...resp.payload } }
   }
+
+  /*
+   * Function to take some Business attributes and update the Business in
+   * Marqeta with these values. The return value will be the updated Business.
+   */
+  async update(business: Partial<Business>): Promise<{
+    success: boolean,
+    body?: Business,
+    error?: MarqetaError,
+  }> {
+    /* eslint-disable no-unused-vars */
+    const {
+      created_time,
+      status,
+      metadata,
+      last_modified_time,
+      ...updateOptions
```

Comment:
As in arrays, all restructuring lines end in a comma - this one too.

Reply:
This does not work, typescript fails to compile with trailing comma:

`buildx@Johns-MBP api % npm run build

> api@0.0.0 build
> npx tsc; cp -r build/* .

src/marqeta/marqeta-node-client/src/business.ts:192:23 - error TS1013: A rest parameter or binding pattern may not have a trailing comma.

192       ...updateOptions,
                          ~


Found 1 error in src/marqeta/marqeta-node-client/src/business.ts:192`

Reply:
buildx@Johns-MBP api % npm run build

> api@0.0.0 build
> npx tsc; cp -r build/* .

src/marqeta/marqeta-node-client/src/business.ts:192:23 - error TS1013: A rest parameter or binding pattern may not have a trailing comma.

192       ...updateOptions,
                          ~


Found 1 error in src/marqeta/marqeta-node-client/src/business.ts:192

Reply:
This is fine - without the comma - due to the `...` function...